Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:10
+WORKDIR /src
+
+RUN echo $(grep "VERSION=" /etc/os-release | cut -d "(" -f2 | cut -d ")" -f1) | \
+  xargs -i echo "deb http://apt.postgresql.org/pub/repos/apt/ {}-pgdg main" > /etc/apt/sources.list.d/postgresql.list && \
+  wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+
+# install PostgreSQL 9.5
+RUN apt-get update && apt-get -y upgrade && \
+  apt-get install --no-install-recommends -y \
+  postgresql-client-9.5 && \
+  apt-get clean
+
+# install Azure Functions Core Tools 3
+RUN apt-get install -y apt-transport-https && \
```

Comment:
this should be broken apart into the apt package setup (like we do in line 4) and put that part above the first `apt-get` cmd on line 9.

then combine the actual `-y azure-functions-core-tools-3` install into the first `apt-get install` on line 9-12. this will help with docker image layer caching and final image sizes.

Reply:
Got it- rearranged.